visual_regression_tracker/exceptions.py

```diff
@@ -0,0 +1,25 @@
+from .types import TestRunStatus
+
+
+class VisualRegressionTrackerError(Exception):
+    """An error occurred."""
+    
+    pass
+
+
+class ServerError(VisualRegressionTrackerError):
+    """An error occurred on the VisualRegressionTracker server."""
+
+    def __init__(self, response: dict, *args):
+        """Initialises ServerError from server response."""
+        self.response = response
+        super(ServerError, self).__init__(*args)
+
+
+class TestFailed(VisualRegressionTrackerError):
+    """A visual test failed."""
+
+    def __init__(self, status: TestRunStatus, *args):
+        """Initialises TestFailed error."""
+        self.status = status
```

Comment:
not sure if I'm getting how it works with `status`
this is the error I'm getting now and I cannot see status there
```
visual_regression_tracker.exceptions.TestRunError: No baseline: http://localhost:8080/4472293f-e07b-4904-8461-c4a34fb0440d?buildId=2718ed4f-3eaa-45df-8236-324dbbe82ded&testId=c4b37d9b-77e3-45dd-8f89-28ee9632dae3
```